fortinet-fortirecon-brand-protection/info.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "fortinet-fortirecon-brand-protection",
   "label": "Fortinet FortiRecon Brand Protection",
-  "version": "2.0.0",
+  "version": "2.1.0",
   "description": "FortiRecon is a Digital Risk Protection Service (DRPS) product that provides an outside-the-network view to the risks posed to your enterprise.",
   "publisher": "Fortinet",
+  "contributor": "Ramkrushna Pathade",
   "cs_approved": true,
   "cs_compatible": true,
   "category": "Attack Surface Management",
```

Comment:
please update the "help_online" key to empty string.
"help_online": ""

Reply:
Done